core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -18,25 +18,26 @@
 package kafka.server
 
 
-import kafka.api.SerializationTestUtils
-import kafka.message.{Message, ByteBufferMessageSet}
-import kafka.utils.{ZkUtils, MockScheduler, MockTime, TestUtils}
-import org.apache.kafka.common.requests.ProduceRequest
-
-import java.util.concurrent.atomic.AtomicBoolean
 import java.io.File
+import java.util.concurrent.atomic.AtomicBoolean
 
+import kafka.cluster.Broker
+import kafka.message.{ByteBufferMessageSet, Message}
+import kafka.utils.{MockScheduler, MockTime, TestUtils, ZkUtils}
+import org.I0Itec.zkclient.ZkClient
 import org.apache.kafka.common.metrics.Metrics
-import org.apache.kafka.common.TopicPartition
 import org.apache.kafka.common.protocol.Errors
+import org.apache.kafka.common.requests.LeaderAndIsrRequest
+import org.apache.kafka.common.requests.LeaderAndIsrRequest.PartitionState
 import org.apache.kafka.common.requests.ProduceResponse.PartitionResponse
 import org.apache.kafka.common.utils.{MockTime => JMockTime}
+import org.apache.kafka.common.{BrokerEndPoint, TopicPartition}
 import org.easymock.EasyMock
-import org.I0Itec.zkclient.ZkClient
+import org.junit.Assert.{assertTrue, assertEquals}
 import org.junit.Test
 
+import scala.collection.JavaConversions._
 import scala.collection.Map
-import scala.collection.JavaConverters._
```

Comment:
`JavaConverters` is recommended over `JavaConversions`. Can you explain why you changed this?